### PR TITLE
Update to MP6 and JEE10

### PR DESCRIPTION
**Pull request by gkwan-ibm**
address issue: https://github.com/OpenLiberty/guides-common/issues/891 and https://github.com/OpenLiberty/guides-common/issues/897

### Review changes
- [ ]  pom.xml
  - java 11
  - jakarta is 10, microfile-profile is 6.0, and dependencies
  - LMP 3.7.1
- [ ] server.xml
  - features version
- [ ] index.html
  - license and features version
- [ ] java files
  - EPL 2.0
- [ ] License file
   - EPL 2.0

### End-to-end test
- [ ] update lgdev with `version-update-mp6` branch
- [ ] do end-to-end test and review content carefully
  - clone the `version-update-mp6` branch
  - if index.html has changes, make sure the links work

### Cloud hosted test
- [ ] generate the cloud hosted dev lab
- [ ] do end-to-end test and review content
  - clone the `version-update-mp6` branch

### CODE REVIEW

File: finish/src/main/liberty/config/server.xml
Comment (jakub-pomykala) [inline finish/src/main/liberty/config/server.xml:1]: Should we not update the copyright since we updating the file? @gkwan-ibm

Comment (gkwan-ibm) [reply finish/src/main/liberty/config/server.xml:1]: you can remove the license header in the server.xml. The server.xml in other guides does not have the license header.

**Comment by jakub-pomykala** [comment]
address issue: https://github.com/OpenLiberty/guides-common/issues/891

### Review changes
- [x]  pom.xml
  - java 11
  - jakarta is 10, microfile-profile is 6.0, and dependencies
  - LMP 3.7.1
- [x] server.xml
  - features version
- [x] index.html
  - license and features version
- [x] java files
  - EPL 2.0
- [x] License file
   - EPL 2.0

### End-to-end test
- [x] update lgdev with `version-update-mp6` branch
- [x] do end-to-end test and review content carefully
  - clone the `version-update-mp6` branch
  - if index.html has changes, make sure the links work

### Cloud hosted test
- [x] generate the cloud hosted dev lab
- [x] do end-to-end test and review content
  - clone the `version-update-mp6` branch

**Review by jakub-pomykala — APPROVED** [review]
Tested locally and on SNL. 

Both run successfully with no issues.

**Pull request merged**